src/pages/Chat/steps/CurrentInsuranceInput.test.tsx

```diff
@@ -12,7 +13,15 @@ it('handles form changes', () => {
     <Provider<WithStorageProps>
       initialState={{ storage: { session: createSession(new MockStorage()) } }}
     >
-      <CurrentInsuranceInput />,
+      <MockTextKeyProvider
```

Comment:
What's the use-case for mocking? :) Shouldn't textkeyfy just return the passed text key? or does it throw due to it not having a provider? (in that case we should just fix that)

Reply:
Well, we could pull in `MockedProvider` from `react-apollo/test-utils` and mock the network request. The problem is that the test depends on the items that the text key replaces, meaning the text key itself isn't good enough. So we need the actual text, not the text key.

Reply:
Ah, ok fine then!